Formula/rtags.rb

```diff
@@ -28,6 +28,45 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
+    The instance of rtags that run via brew services listens to
+    the socket file at #{var}/.rdm
```

Comment:
Great!

I need some help with filling out this part:
```
<dict>
  <key>SockPathName</key>
  <string>#{var}/.rdm</string>
</dict>
```
What can I replace `#{var}` with, to get the user's `$HOME` directory?